node/src/components/rpc_server/rpcs/state.rs

```diff
@@ -558,3 +576,262 @@ impl RpcWithParamsExt for GetAccountInfo {
+                        Err(_) | Ok(None) => {
+                            return Ok(response_builder.error(warp_json_rpc::Error::custom(
+                                ErrorCode::NoDictionaryName as i64,
+                                "Failed to execute query",
+                            ))?)
+                        }
+                    };
+
+                    let query_result = effect_builder
+                        .make_request(
+                            |responder| RpcRequest::QueryGlobalState {
+                                state_root_hash: params.state_root_hash,
+                                base_key,
+                                path,
+                                responder,
+                            },
+                            QueueKind::Api,
+                        )
+                        .await;
+
+                    let (stored_value, _) = match common::extract_query_result(query_result) {
+                        Ok(tuple) => tuple,
+                        Err((error_code, error_msg)) => {
+                            info!("{}", error_msg);
+                            return Ok(response_builder.error(warp_json_rpc::Error::custom(
+                                error_code as i64,
+                                error_msg,
+                            ))?);
+                        }
+                    };
+
+                    params
+                        .dictionary_identifier
+                        .get_dictionary_address(Some(stored_value))
+                }
+                DictionaryIdentifier::URef { .. } => {
+                    params.dictionary_identifier.get_dictionary_address(None)
+                }
+                DictionaryIdentifier::Dictionary { .. } => {
+                    params.dictionary_identifier.get_dictionary_address(None)
+                }
+            };
+
+            let dictionary_query_key = match dictionary_address {
+                Ok(key) => key,
+                Err(Error(message)) => {
+                    return Ok(response_builder.error(warp_json_rpc::Error::custom(
+                        ErrorCode::FailedToGetDictionaryURef as i64,
+                        message,
+                    ))?)
+                }
+            };
+
+            let query_result = effect_builder
+                .make_request(
+                    |responder| RpcRequest::QueryGlobalState {
+                        state_root_hash: params.state_root_hash,
+                        base_key: dictionary_query_key,
+                        path: vec![],
+                        responder,
+                    },
+                    QueueKind::Api,
+                )
+                .await;
+
+            let (stored_value, proof_bytes) = match common::extract_query_result(query_result) {
+                Ok(tuple) => tuple,
+                Err((error_code, error_msg)) => {
+                    info!("{}", error_msg);
```

Comment:
```suggestion
                    error!("{}", error_msg);
```

Reply:
Might depend on the code if this is a system error or a user error...

Reply:
Yeah, Im going to keep it as `info`

Reply:
It seems to have been `info` across the other functions, so I shall retain it as such